lighttpd_rust_amalgamated/Cargo.toml

```diff
@@ -9,13 +9,9 @@ publish = false
 edition = "2021"
 autobins = false
```

Comment:
Also remove this if we remove `[[bin]]`.

Reply:
62b3eb0